pkgs/by-name/ru/ruff/package.nix

```diff
@@ -4,16 +4,20 @@
   fetchFromGitHub,
   installShellFiles,
   stdenv,
+  python3Packages,
   darwin,
   rust-jemalloc-sys,
   ruff-lsp,
   nix-update-script,
   versionCheckHook,
 }:
 
-rustPlatform.buildRustPackage rec {
+python3Packages.buildPythonPackage rec {
   pname = "ruff";
   version = "0.7.0";
+  format = "pyproject";
+
+  disabled = python3Packages.pythonOlder "3.7";
```

Comment:
```suggestion
```

This is not that helpful, especially if it's 3.7.

Reply:
habits